### PR TITLE
New Rapier for traitors - costs 12 trilby cats

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Adds a new rapier, and its shift for 12 tc
Rapier does 100 armor pen, but still can be blocked
33 harm 5 hits to soft crit
45 throw harm
Can not block anything as its a rapier
Sheaf -
5 harm 
15 throw
30 block on all melee - 0 on range 
## Why It's Good For The Game

A way cooler non-energy sword that is meant for loud traitors

## Changelog
:cl:
add: rapier and such
/:cl:
These weapons are like capo saber and are noticeable at a glace, and mega valid saild

### CODE REVIEW

File: code/game/objects/items/melee/misc.dm [hunk 1]
```diff
@@ -133,6 +133,28 @@
...
+/obj/item/melee/rapier
+	name = "plastitanium rapier"
+	desc = "A impossibly thin blade made of plastitanium with a tip made of diamond. It looks to be able to cut through any armor."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "rapier"
+	item_state = "rapier"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	force = 33
+	throwforce = 45
```
Comment (Poojawa) [inline code/game/objects/items/melee/misc.dm:145]: this + 100 AP means you just pair it with TK in crit people in 3 clicks.

Comment (Useroth) [reply code/game/objects/items/melee/misc.dm]: _33*3 = 99 tho,
also, on Cita, you aren't in crit until you get 120 damage _

Comment (Useroth) [reply code/game/objects/items/melee/misc.dm:145]: Hm... it should probably have a good chance to embed.

Comment (Trilbyspaceclone) [reply code/game/objects/items/melee/misc.dm:145]: No

**Comment by silicons** [comment]
33 harm is 4 hits to soft crit we already have eswords sorry this isn't necessary

**Pull request closed without merging**